src/OpenTelemetry/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Diagnostics.Metrics;
+using System.Reflection;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using Temporalio.Client;
+using Temporalio.Extensions.DiagnosticSource;
+using Temporalio.Extensions.OpenTelemetry;
+using Temporalio.Runtime;
+using Temporalio.Worker;
+using TemporalioSamples.OpenTelemetry;
+
+AssemblyName assemblyName = typeof(TemporalClient).Assembly.GetName();
```

Comment:
```suggestion
var assemblyName = typeof(TemporalClient).Assembly.GetName();
```

Feel free to use `var` throughout this file